src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterRepository.java

```diff
@@ -120,6 +122,20 @@ public int updateIsFailed(UUID id) {
         );
     }
 
+    /**
+     * Retrieve a list of letters which are not failed but still not printed.
+     *
+     * @return list of not printed letters until yesterday's 5pm
```

Comment:
judging by the query it's a list of letters not printed until now, that were created before yesterday 5pm.

Reply:
* that were sent to print yesterday before 5pm